.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: elixir
 elixir:
+  - 1.3.1
```

Comment:
It was 1.3.1 before (https://github.com/sorentwo/braintree-elixir/blob/ca185e8b1d46953a80f725e38d36ea7a21f7e965/.travis.yml#L3), so I just added that again. I'm guessing that there aren't any specific reasons right now why this library couldn't support 1.3.1 too.